pkgs/by-name/wf/wfview/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  kdePackages,
+  eigen,
+  hidapi,
+  libopus,
+  libpulseaudio,
+  portaudio,
+  rtaudio,
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "wfview";
+  version = "2.10";
+
+  src = fetchFromGitLab {
+    owner = "eliggett";
+    repo = "wfview";
+    rev = "v${finalAttr.version}";
+    hash = "sha256-bFTblsDtFAakbSJfSfKgvoxd1DTSv++rxU6R3/uWo+4=";
+  };
+
+  patches = [
+    # Remove syscalls during build to make it reproducible
+    # We also need to adjust some header paths for darwin
+    ./remove-hard-encodings.patch
+  ];
+
+  buildInputs = (
+    [
+      eigen
+      hidapi
+      libopus
+      libpulseaudio
+      portaudio
+      rtaudio
+    ]
+    ++ (with kdePackages; [
```

Comment:
A small nitpick: pedantically, libraries coming from a Qt 6 installation are in `qt6` (e.g. `qt6.qtbase`), while `qcustomplot` is in `qt6Packages`.